modules/ROOT/pages/bare-metal.adoc

```diff
@@ -0,0 +1,98 @@
+= Installing Fedora bootc on Bare Metal
+
+This guide provides instructions to install Fedora bootc to bare metal.
+Three main options are available:
+
+* Installing from a stock Anaconda ISO/PXE over the network
+* Installing from a bootc-image-builder generated ISO
+* Installing from the container directly with `bootc install`
+
+== Prerequisites
+
+Before installing Fedora bootc, it's recommended that you have created a customized derived container image;
+but this is not a hard requirement, as it is possible to enable basic system access via
+e.g. injecting SSH keys with kickstart or with `bootc install` and the `-root-ssh-authorized-keys`
```

Comment:
```suggestion
e.g. injecting SSH keys with kickstart or with `bootc install` and the `--root-ssh-authorized-keys`
```

though this option doesn't seem to exist in F40 image:

```
[vagrant@vanilla-rawhide ~]$ sudo podman run --rm --privileged --pid=host -v /var/lib/containers:/var/lib/containers --security-opt label=type:unconfined_t registry.gitlab.com/bootc-org/fedora-bootc/base-images-experimental/fedora-bootc-full:40 bootc install to-disk /dev/vda --root-ssh-authorized-keys /var/lib/containers/authorized_keys 
error: unexpected argument '--root-ssh-authorized-keys' found

  tip: a similar argument exists: '--root-size'

Usage: bootc install to-disk <DEVICE|--wipe|--block-setup <BLOCK_SETUP>|--filesystem <FILESYSTEM>|--root-size <ROOT_SIZE>>

For more information, try '--help'.
```

Reply:
https://bodhi.fedoraproject.org/updates/FEDORA-2024-cf546a8d55

Reply:
sounds good.